packages/libs/react-components/src/components/Hr/index.ts

```diff
@@ -0,0 +1,10 @@
+import { styled } from '../../styles';
+
+import { StyledComponent } from '@stitches/react/types/styled-component';
+
+export const Hr: StyledComponent<'hr'> = styled('hr', {
```

Comment:
I think `Divider` would be a more descriptive name

Reply:
done..sounds good